src/stream_base-inl.h

```diff
@@ -278,7 +278,7 @@ void WriteWrap::SetBackingStore(std::unique_ptr<v8::BackingStore> bs) {
   backing_store_ = std::move(bs);
 }
 
-void StreamReq::Done(int status, const char* error_str) {
+v8::Maybe<void> StreamReq::Done(int status, const char* error_str) {
```

Comment:
We should also change all call sites of the functions to do the needful based on the return values. I think it would be better if we do this on a per-function basis to keep the PR small and easier to review. :)

Reply:
I've cleaned up this PR only for `OnUvRead()`. :-)